R/pkg/R/backend.R

```diff
@@ -25,9 +25,23 @@ isInstanceOf <- function(jobj, className) {
   callJMethod(cls, "isInstance", jobj)
 }
 
-# Call a Java method named methodName on the object
-# specified by objId. objId should be a "jobj" returned
-# from the SparkRBackend.
+#' Call Java Methods
+#'
+#' Call a Java method in the JVM running the Spark driver.
+#'
+#' @param objId object to invoke the method on. Should be a "jobj" created by newJObject.
+#' @param methodName method name to call.
```

Comment:
`@param methodName character method name to call. this should be a Java fully qualified class name.`